pandas/tests/indexing/test_iloc.py

```diff
@@ -1148,6 +1148,18 @@ def test_loc_setitem_boolean_list(self, rhs_func, indexing_func):
         expected = DataFrame({"a": [5, 1, 10]})
         tm.assert_frame_equal(df, expected)
 
+    def test_iloc_getitem_slice_negative_step_ea_block(self):
+        # GH#44551
```

Comment:
Is an additional Test for loc necessary?

Reply:
not necessary, but harmless.  will update.

Reply:
i take it back, will be a bit of a hassle to test loc, so leaving as-is